README.md

```diff
@@ -1,2 +1,113 @@
+<div align="center">
+<h1>CDK Data Availability</h1>
+<h3>Data Availability Layer for CDK Validium</h3>
+
+</div>
+
+<p align="left">
+  The cdk-data-availability project is a specialized Data Availability Node (DA Node) that is part of Polygon's CDK (Chain Development Kit) Validium.
+</p>
+
+<!-- TOC -->
+
+- [Overview of Validium](#overview-of-validium)
+- [Introduction](#introduction)
+- [Key Components](#key-components)
+  * [Off-Chain Data](#off-chain-data)
+  * [Data Availability Committee](#data-availability-committee)
+  * [Sequencer](#sequencer)
+  * [Synchronizer](#synchronizer)
+- [Prerequisites](#prerequisites)
+- [Deployment](#deployment)
+- [License](#license)
+
+## Overview of Validium
+
+For a full overview of the Polygon CDK Validium, please reference the [CDK documentation](https://wiki.polygon.technology/docs/cdk/).
+
+The CDK Validium solution is made up of several components; start with the [CDK Validium Node](https://github.com/0xPolygon/cdk-validium-node). For quick reference, the complete list of components are outlined below:
+
+| Component                                                                     | Description                                                          |
+| ----------------------------------------------------------------------------- | -------------------------------------------------------------------- |
+| [CDK Validium Node](https://github.com/0xPolygon/cdk-validium-node)           | Node implementation for the CDK networks in Validium mode            |
+| [CDK Validium Contracts](https://github.com/0xPolygon/cdk-validium-contracts) | Smart contract implementation for the CDK networks in Validium mode |
+| [CDK Data Availability](https://github.com/0xPolygon/cdk-data-availability)   | Data availability implementation for the CDK networks          |
+| [Prover / Executor](https://github.com/0xPolygonHermez/zkevm-prover)          | zkEVM engine and prover implementation                               |
+| [Bridge Service](https://github.com/0xPolygonHermez/zkevm-bridge-service)     | Bridge service implementation for CDK networks                       |
+| [Bridge UI](https://github.com/0xPolygonHermez/zkevm-bridge-ui)               | UI for the CDK networks bridge                                       |
+
+---
+
+## Introduction
+
+As blockchain networks grow, the volume of data that needs to be stored and validated increases, posing challenges in scalability and efficiency. Storing all data on-chain can lead to bloated blockchains, slow transactions, and high fees.
+
+Data Availability Nodes facilitate a separation between transaction execution and data storage. They allow transaction data to reside off-chain while remaining accessible for validation. This significantly improves scalability and reduces costs. Within the framework of Polygon's CDK, the DA Node is managed by a Data Availability Committee (DAC) to ensure the security, accessibility, and reliability of off-chain data.
```

Comment:
> the DA Node is managed by a Data Availability Committee (DAC) to ensure the security

not sure how to phrase that, but a DAC is composed by members, each member should run a DA Node. The way it's written, it looks like there is a single DA NOde managged by a sort of DAO

Reply:
Updated.